CMakeLists.txt

```diff
@@ -62,6 +62,9 @@ if(NOT WIN32)
 endif()
 string(REPLACE ";" " " CMAKE_CXX_FLAGS "${CMAKE_CXX_FLAGS}")
 
+include_directories(${USD_INCLUDE_DIR})
```

Comment:
Using `include_directories` in not recommended since it operates on the directory level and all targets defined on that level inherit those properties. This makes it hard to see what target really needs these dependencies. The better approach is to operate on the targets directly. Please use `target_include_directories`

Reply:
Ok will do - thanks